### PR TITLE
Removed Commented Code

### PR DESCRIPTION
**Pull request by sajdakabir**
## Related Issue
  - Info about Issue or bug

Closes: #887

#### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Please let us know if any test cases are added

Please describe the tests(if any). Provide instructions how its affecting the coverage.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Comment by sajdakabir** [comment]
I have read the CLA Document and I hereby sign the CLA

File: cmd/root.go [hunk 1]
```diff
@@ -54,19 +51,12 @@ func customTimeEncoder(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
...
-// 	enc.AppendString(emoji + " " + level.CapitalString())
-// }
-
 func newRoot() *Root {
 	return &Root{
 		subCommands: []Plugins{},
 	}
 }
 
-// Execute adds all child commands to the root command.
```
Comment (re-Tick) [inline cmd/root.go:68]: Hi @sajdakabir, comment at line 68 and 69 is the description of execute function, so please don't remove this comments.

File: pkg/models/mock.go [hunk 2]
```diff
@@ -16,11 +16,7 @@ type MockSpec struct {
 	HttpReq  *HttpReq  `json:"Req,omitempty"`
 	HttpResp *HttpResp `json:"Res,omitempty"`
 	Created  int64     `json:"Created,omitempty"`
-	// for MongoSpec
```
Comment (re-Tick) [inline pkg/models/mock.go:19]: Please don't remove the descriptive comments.

**Comment by re-Tick** [comment]
Please sign the CLA and DCO.

**Pull request closed without merging**